Disable AltiVec for it breaks the build due to various undefined symbols (vec_sqrt, etc)

Index: vendor/eigen/Eigen/src/Core/util/ConfigureVectorization.h
--- vendor/eigen/Eigen/src/Core/util/ConfigureVectorization.h.orig
+++ vendor/eigen/Eigen/src/Core/util/ConfigureVectorization.h
@@ -373,7 +373,7 @@
     #undef vector
     #undef pixel
 
-  #elif defined __ALTIVEC__
+  #elif defined __ALTIVEC__ && !defined(__OpenBSD__)
 
     #define EIGEN_VECTORIZE
     #define EIGEN_VECTORIZE_ALTIVEC
